package.json

```diff
@@ -63,13 +63,9 @@
     "clean:full": "node ./tools/build/clean-repo.js && lerna clean --yes && lerna run clean && rm -rf node_modules/ && yarn",
     "clean:core": "yarn workspace @twilio-paste/core clean",
     "pre-test": "node ./tools/build/pre-test.js",
-    "test": "yarn pre-test && concurrently \"yarn:test:packages\" \"yarn:test:tools\"",
-    "test:react-16": "yarn pre-test && concurrently \"yarn:test:packages-16\" \"yarn:test:tools-16\"",
-    "test:packages": "jest --silent ./packages/",
-    "test:packages-16": "USE_REACT_16=true jest --silent ./packages/",
-    "test:tools": "jest ./tools/ -c ./tools/jest.config.js",
-    "test:tools-16": "USE_REACT_16=true jest ./tools/ -c ./tools/jest.config.js",
-    "test:coverage": "yarn pre-test && jest --coverage",
+    "test": "jest",
+    "test:react-16": "USE_REACT_16=true jest",
+    "test:coverage": "jest --coverage",
```

Comment:
No need to run pre-test with yarn 3, and no need to have two concurrent jest just to test packages which are tsx and tools which are js.

Reply:
LOVE

Reply:
Also love that silent is removed.. I delete that on my local all the time and then have to put it back lol

Reply:
Totally! 🪙 🪙 Silent _can_ be nice in CI, helps to make output cleaner, but can def obfuscate issues when they arise. So glad we made these improvements. 🏅